whitebox/streamlit/mock/performance.json

```diff
@@ -0,0 +1,131 @@
+[
+    {
+        "id": "b4bff1cd-76bd-47e6-8446-4ce690096dac",
+        "created_at": "2023-02-09T14:33:00.473312",
+        "updated_at": "2023-02-09T14:33:00.473312",
+        "accuracy": 0.6,
+        "precision": {
+            "micro": 0.6,
+            "macro": 0.7333333333333334,
+            "weighted": 0.9199999999999999
+        },
+        "recall": {
```

Comment:
you should type these because the next dev is going to have a bad time debugging